introduction.md

```diff
@@ -1 +1,3 @@
 Hello, my name is Dana! My favorite Code School Path is HTML & CSS.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?